core/templates/dev/head/services/MessengerService.js

```diff
@@ -154,7 +154,6 @@ oppia.factory('MessengerService', ['$log', '$window', function($log, $window) {
           $window.parent.postMessage(JSON.stringify(objToSendToParent), '*');
         } else {
           $log.error('Unknown version for embedding: ' + hashDict.version);
-          return;
```

Comment:
I don't think this warrants automatic removal, generally speaking. What if there's another if clause after this one? Returning and not returning would have different effects. Is the linter smart enough to detect that?

Reply:
@seanlip: I think it does. I tried out the following code locally:  
```javascript
var testVar = 5;
var anotherTestVar = 6;

if (testVar === 5) {
  console.log('inside first if condition')
  return;
}
if (anotherTestVar === 6') {
  console.log('inside second if condition')
  return;
  }
```
It does not show any error. Also, the feature was discussed [here](https://github.com/eslint/eslint/issues/7309). This type of false positive issue was discussed [here](https://github.com/eslint/eslint/issues/7477) and fixed [here](https://github.com/eslint/eslint/pull/7482). Is there anything else that we need to look for?

Reply:
Ah, thanks for linking to the documentation.

I've thought about it, and here's what I suggest -- let's not do it. It feels "too smart" to me. I think it would be better to restrict our lint fixes to issues that actually come up during the course of a code review in Oppia, and I haven't really seen this one, I think.

Reply:
That sounds good. Agreed! Closing this PR. 
Thanks!